### DISCUSSION TITLE
IOException: Cannot locate resource 'themes/recommended/accent/materialdesigncolor.lime.xaml'.

### DISCUSSION THREAD

**Discussion by CallMeDang123** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3576] [Q&A] [answered]
<Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Light.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign2.Defaults.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Primary/MaterialDesignColor.DeepPurple.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Accent/MaterialDesignColor.Lime.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>

**Comment by Keboo** [comment, marked as answer]
This was one of the [breaking changes with the 5.0.0 release](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2255). The "Accent" (near the end of the path in your last resource dictionary), and been renamed to be "Secondary".

**Reply by CallMeDang123** [reply]
thank you <3

**Reply by KothariMansi** [reply]
Thanks. I was struggling with it too.

**Comment by nguyenductai29** [comment]
IOException: Cannot locate resource 'themes/recommended/accent/materialdesigncolor.blue.xaml'

**Comment by TravisGao** [comment]
Thanks very much, it works
